kivy/uix/behaviors/focus.py

```diff
@@ -531,7 +531,9 @@ def keyboard_on_key_down(self, window, keycode, text, modifiers):
         key was consumed.
         '''
         if keycode[1] == 'tab':  # deal with cycle
-            if ['shift'] == modifiers:
+            if {'ctrl', 'alt', 'meta', 'super', 'compose'} & modifiers:
+                return false
+            if {'shift'} & modifiers:  # or "in"
```

Comment:
```suggestion
                return False
            if 'shift' in modifiers:
```

It's probably faster to use `in`. And False was not capitalized.